_examples/basic/2-realtime-feed/README.md

```diff
@@ -0,0 +1,34 @@
+# Realtime Feed
+
+This example features a very busy blogging platform, with thousands of messages showing up on your feed.
+
+There are two separate applications (microservices) integrating over a Kafka topic. The [`producer`](producer/) generates
+thousands of "posts" and publishes them to the topic. The [`consumer`](consumer/) subscribes to this topic and 
+displays each post on the standard output.
+
+The consumer has a throttling middleware enabled, so you have a chance to actually read the posts.
+
+To understand the background and internals, see [getting started guide](https://watermill.io/docs/getting-started/).
+
+## Requirements
+
+To run this example you will need Docker and docker-compose installed. See the [installation guide](https://docs.docker.com/compose/install/).
+
+## Running
+
+```bash
+docker-compose up
+```
+
+You should see the live feed of posts on the standard output.
+
+## Exercises
+
+1. Peek into the posts counter published on `posts_count` topic.
+
+```
+docker-compose exec kafka kafka-console-consumer --bootstrap-server localhost:9092 --topic posts_count
```

Comment:
In theory, we have mill ;) but it's hard to run it with the one-liner, maybe for the future ;)

Reply:
Nice idea, I'll consider adding a note in some way. I don't think we have mill covered anywhere in README or in the docs.

Reply:
So we could add another service to the compose, but it gets kinda ugly running it inside compose, with dependencies and mounting volumes by relative path. It works, but I feel like using kafka-console-consumer can be a bit easier to understand by newcomers.

Instead, I think we could add a separate docs page for mill. What do you think?

Reply:
but maybe just mill without docker, and link to sub-directory (to have info how to install etc)

Reply:
but maybe in another task :P added to todolist